KerbalStuff/antivirus.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+import pyclamd
+
+from .config import _cfg, _cfgi, site_logger
+from .objects import User, Mod
+from .email import send_mod_locked
+from .ckan import notify_ckan
+
+clam_daemon = None
+
+
+def file_contains_malware(where: str) -> bool:
+    global clam_daemon
+    try:
+        if not clam_daemon:
+            clam_daemon = pyclamd.ClamdNetworkSocket(host=_cfg('clamav-host'), port=_cfgi('clamav-port', 3310))
+        result = clam_daemon.scan_file(where)
+        if result:
+            site_logger.error(f'ClamAV says {where} contains malware')
+            return True
+    except Exception as exc:
+        # No ClamAV daemon found, log it and let the file through
+        site_logger.error(f'Failed to connect to ClamAV, skipping scan of {where}', exc_info=exc)
+    return False
+
+
+def punish_malware(user: User, mod: Optional[Mod] = None) -> None:
```

Comment:
The `mod` argument is unused

Reply:
Huh, I must have thought that the problematic mod would have been important, then realized that we won't have a `Mod` object if we abort creation, then not gone back and refactored it. Removed.